packages/cxl-lumo-styles/scss/themes/cxl-tabs-slider.scss

```diff
@@ -26,6 +26,8 @@
   // Fading content doesn't work well on slider.
   --_lumo-tabs-overflow-mask-image: unset !important; /* stylelint-disable-line declaration-no-important */
 
+  overflow: auto;
```

Comment:
Also clarify w/ `// Firefox.` or?

This has no negative effects on webkit browsers? What's the underlying necessity about?

Reply:
This is set to `overflow: hidden` by vaadin-tabs. If this isn't overridden, Firefox won't display a scrollbar.

I don't see any regression on Webkit browsers.